### DISCUSSION TITLE
Running a workflow which requires tag push

### DISCUSSION THREAD

**Discussion by ahmubashshir** [nektos/act#1023] [Q&A] [answered]
In the workflow I'm testing, it has a step with `if: ${{ startsWith(github.ref, 'refs/tags/v') }}`
which is always skipped, how can I check this step?

**Comment by catthehacker** [comment, marked as answer]
Use event file https://github.com/nektos/act#events

**Reply by ahmubashshir** [reply]
I know, but how would I write event file for tags?

Relevant issue: #179

**Reply by catthehacker** [reply]
> I know, but how would I write event file for tags?

Type it? https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

**Reply by catthehacker** [reply]
https://github.com/nektos/act/wiki/Beginner's-guide#using-event-file-to-provide-complete-event-payload

**Reply by taras** [reply]
The above link does not work anymore. It looks like it might be this page now https://nektosact.com/usage/index.html
